### PR TITLE
Lectures/alfredo zimperz/tour of heroes redux

### PR DESCRIPTION
**Pull request by fredyzz**

### CODE REVIEW

File: alfredo-zimperz/lectures/tour-of-heroes-redux/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
```
Comment (edalandete) [inline alfredo-zimperz/lectures/tour-of-heroes-redux/.eslintrc.js:6]: Good job starting with test!

File: alfredo-zimperz/lectures/tour-of-heroes-redux/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Tour of Heroes</title>
```
Comment (edalandete) [inline alfredo-zimperz/lectures/tour-of-heroes-redux/public/index.html:27]: Good changing the title of the app

File: alfredo-zimperz/lectures/tour-of-heroes-redux/src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import Dashboard from './components/Dashboard';
+import Heroes from './components/Heroes';
+import NotFound from './components/NotFound';
+import HeroDetail from './components/HeroDetail';
+import './styles.css';
+import './App.css';
+
+function App() {
+  return (
+    <>
```
Comment (edalandete) [inline alfredo-zimperz/lectures/tour-of-heroes-redux/src/App.jsx:16]: you can remove the <> and </>

File: alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import HEROES from '../../constants/heroes.mock';
+
+export function loadHeroes() {
+  return {
+    type: actionTypes.LOAD_HEROES,
+  };
+}
+
+export function addHero(heroName) {
+  const nextId = HEROES.length;
```
Comment (edalandete) [inline alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js:11]: i think you will be adding the same id to your heroes.

**Review by edalandete — APPROVED** [review]

**Review by annasm07 — APPROVED** [review]

**Review by cristian-bars — APPROVED** [review]
Good job!

File: alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js (hunk 4 above)
Comment (davzlum) [reply alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js:11]: I agree, its creating always the same Id :/

**Review by jasuarez21 — APPROVED** [review]

File: alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js (hunk 4 above)
Comment (fredyzz) [reply alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js]: Fixed, thank you guys!

**Review by jasuarez21 — APPROVED** [review]

**Review by edalandete — APPROVED** [review]
Molt bé tiu

**Review by jasuarez21 — APPROVED** [review]
listo

**Review by davzlum — APPROVED** [review]
Grande Freddy!

**Review by IrenePerezAlonso — APPROVED** [review]

**Pull request merged**